lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -2552,8 +2983,7 @@ int s1ap_handle_paging_request(
     }
   }
   free_wrapper((void**) &enb_array->elements);
-  free_wrapper((void**) &enb_array);
```

Comment:
@Vyas20 Freeing of enb array elements is needed. Removal of this line is leading to memory leaks for "idle mode multi pdn" TCs . Please make sure that the fixes done in PR https://github.com/magma/magma/pull/2875 do not get modified in your code. Attaching memory leaks I found in this code base on running "idle mode multi pdn". You can verify these TCs from PRs https://github.com/magma/magma/pull/2521 and https://github.com/magma/magma/pull/2758
![memory_leak_idle_mode_multi_pdn](https://user-images.githubusercontent.com/38202471/101135144-6aeac400-3631-11eb-8bd0-d94b5b82c0f4.PNG)